.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Dependency Review'
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+    - name: 'Checkout Repository'
+      uses: actions/checkout@v4
+    - name: Dependency Review
+      uses: actions/dependency-review-action@v4
+      with:
+        config-file: './.github/dependency-review-config.yml'
```

Comment:
We might want to consider pointing this to an external reference at some point via something like this:

```
# Syntax for an external file: OWNER/REPOSITORY/FILENAME@BRANCH
config-file: 'wasmcloud/wasmcloud-operator/.github/dependency-review-config.yml@main'
```

But for now I think it should be fine as it is.